environment.yml

```diff
@@ -1,34 +1,37 @@
 name: snorkeling
+channels:
+  - conda-forge
 dependencies:
+- backports.lzma==0.0.8
 - beautifulsoup4=4.6.0
-- enum34=1.1.6
-- future=0.16.0
-- futures=3.1.1
-- ipywidgets=6.0.0
+- ipywidgets=7.1.0
 - jupyter=1.0.0
 - jupyter_client=5.0.1
-- jupyter_console=5.1.0
-- jupyter_core=4.3.0
-- llvmlite=0.18.0
-- matplotlib=2.0.2
+- jupyter_console=5.2.0
+- jupyter_core=4.4.0
+- llvmlite=0.21.0
+- lxml==4.1.1
+- matplotlib=2.1.2
+- neo4j-python-driver==1.3.1
 - nltk=3.2.4
-- numba=0.33.0
-- numpy=1.12.1 
-- pandas=0.20.2 
-- psycopg2=2.7.1
-- requests=2.14.2
-- seaborn=0.7.1
-- scikit-image=0.13.0
-- scikit-learn=0.18.1
-- scipy=0.19.0 
-- six=1.10.0
-- sqlalchemy=1.1.11
-- spacy=1.8.2
-- tqdm=4.14.0
-- conda-forge::tika=1.14.1
+- numba=0.36.2
+- numpy=1.13.0
+- pandas=0.22.0
+- psycopg2=2.7.3.2
+- python=3.6.2
+- py4j=0.10.6
+- requests=2.18.4
+- seaborn=0.8.1
+- scikit-image=0.13.1
+- scikit-learn=0.19.1
+- scipy=1.0.0 
+- six=1.11.0
+- sqlalchemy=1.1.13
+- sqlite=3.20.1
+- spacy=2.0.4
+- tensorflow==1.4.0
+- tqdm=4.19.5
+- tika=1.15
 - pip:
-    - backports.lzma==0.0.8
     - git+https://github.com/HazyResearch/numbskull@ac52265038bac8edca3f8e930eff34ebaef4c7a0
```

Comment:
This should be upgraded to latest probably? https://github.com/HazyResearch/numbskull/commit/52192383b30f2835f5b1e17c6365e2f8c93d97da